### ISSUE TITLE
Method to show AMT connected devices as offline when they are powered off

### ISSUE THREAD

**Issue by unguzov** [Ylianst/MeshCentral#4642]
Problem with "online" status:
I have a problem when I see list full with online devices, but some of them are just AMT available and powered off.

Solution:
1. Option in config file to mark all powered off devices with AMT as "Offline" (gray) .
2. Even better - option for two icons - one for the Desktop agent status and one for AMT status.

**Comment by Ylianst** [comment]
For any device MeshCentral keeps a list of connections it has to this device (Agent, AMT, MQTT...) and the best indication it has of the device's power state (on, sleeping, off, present).

Two icons for each device is starting to get complicated, also, when you filter or sort by "online" or "offline", not sure what I need to do in cases where the agent is offline and Intel AMT is online.

Let me know if your request is just to change the web UI presentation, if so, I will work on adding an option for that. Can the option be in the config.json, or should it be per-user in the web ui?

**Comment by unguzov** [comment]
Hello @Ylianst, thank you for your quick answer! Here is more information about my case:

Most of the time I use "My Devices" and "List" mode to manage our clients. Filter by group ("g:") is extremely useful. In the "List" mode I can see immediately state of devices that are powered on and online and take action from there.

This works very well until group has "Intel AMT" settings and devices activated. All devices with active AMT connection (but powered off) are shown in the list with the same icon as powered on devices. In this case I click on the device just to find out that Desktop/Connect button is gray. Practical use for this state is when we shutdown group of computers and wait until all of them change icon to gray (but this never happen with AMT connected devices). Now we are using column "Connectivity" for Agent/CIRA state, but this is not easy to use in long lists.

I understand that device is technically "online" but I need somehow to distinct between "online because AMT is connected" and "online because PC is powered ON" in the list.

To summarize I will be happy with any of these solutions for the devices list:
- Different icon color for powered on and AMT only connected devices (that are powered off) - for example you are using different colors in "7 Day Power State" for "powered" and "present" and this is great.
- Option in config file (global or per user in web UI) to show device icon as gray in "My Devices/List" in case when AMT is connected, but device is powered off.

I agree that solution with two icons is confusing.

**Comment by si458** [comment]
this already exists now apart from the icons im afraid, 
the power state, shows different colours for different states
https://ylianst.github.io/MeshCentral/meshcentral/devicetabs/#7-day-power-state
